frontend/lib/pages/fee-waiver.tsx

```diff
@@ -100,7 +100,7 @@ export const FeeWaiverExpenses = stepBuilder.createStep({
   </>,
   renderForm: ctx => <>
     <CurrencyFormField
-      label="How much do you pay in rent each month?"
+      label="How much do you pay in rent each month? If you live with someone else, please put in only the part of the rent that you pay."
```

Comment:
So this is also mentioned in the intro of the page, do you want it removed from there?  We also have the ability to put it in as a the hint text for a field, in which case it will be displayed below the field, un-bolded (I was worried that there might be too much bold text with it as part of the label).

Reply:
now i see it. sorry, it wasn't on the priority guide